azplugins/mpcd.py

```diff
@@ -23,6 +25,54 @@
 
 from . import _azplugins
 
+class compute_velocity(hoomd.compute._compute):
+    r"""Compute center-of-mass velocity of MPCD particles
+
+    Args:
+        suffix (str): Suffix to attach to logged quantities.
+
+    This computes the center-of-mass velocity of all MPCD particles:
+
+    .. math::
+
+        \mathbf{v}_{\rm cm} = \dfrac{\sum_i m_i \mathbf{v}_i}{\sum_i m_i}
+
+    where :math:`\mathbf{v}_i` is the velocity and and :math:`m_i` is the mass
+    of particle *i* in the group. Note that because all MPCD particles currently
+    have the same mass, this is equivalent to the number-average velocity.
+
+    The components of the result are exposed as loggable quantities ``mpcd_vx``,
+    ``mpcd_vy``, and ``mpcd_vz`` with ``suffix`` appended. By default,
+    ``suffix`` is an empty string, but a custom suffix may be specified if needed
+    to distinguish the logged quantity from something else. You can save these
+    results using :py:class:`hoomd.analyze.log`.
+
+    Example::
+
+        azplugins.mpcd.compute_velocity()
+        hoomd.analyze.log(filename='velocity.dat', quantities=['mpcd_vx'], period=10)
```

Comment:
same comment as above, maybe add an example with a custom name just for the sake of completeness.